infrastructure/code/snyk_code_http_client.go

```diff
@@ -313,11 +310,13 @@ func (s *SnykCodeHTTPClient) ExtendBundle(
 	removedFiles []string,
 ) (string, []string, error) {
 	method := "code.ExtendBundle"
-	s.c.Logger().Debug().Str("method", method).Msg("API: Extending bundle for " + strconv.Itoa(len(files)) + " files")
-	defer s.c.Logger().Debug().Str("method", method).Msg("API: Extend done")
-
 	span := s.instrumentor.StartSpan(ctx, method)
 	defer s.instrumentor.Finish(span)
+	requestId, _ := performance2.GetTraceId(span.Context())
```

Comment:
This error should be checked. If it shouldn't fail the method, it should at least be logged for diagnostic / debugging purposes.

There's a golangci-lint rule we should turn on that would catch this (for a followup!)

Reply:
We can log it - but it should not fail anything - if this causes an error, it's nothing we can or should do about. I'd rather remove the error return value for `GetTraceId`